airflow/providers/asana/example_dags/example_asana.py

```diff
@@ -31,46 +33,62 @@
     "owner": "airflow",
 }
 
+ASANA_TASK_TO_UPDATE = os.environ.get("ASANA_TASK_TO_UPDATE")
+ASANA_TASK_TO_DELETE = os.environ.get("ASANA_TASK_TO_DELETE")
+ASANA_PROJECT_ID = os.environ.get("ASANA_PROJECT_ID")
+CONN_ID = os.environ.get("ASANA_CONNECTION_ID")
```

Comment:
Is `None` an acceptable value for any of these? If not, it’s better to use `os.environ["ASANA_TASK_TO_UPDATE"]` etc. instead.

Reply:
Oh good point -- done! (that also reminded me to add some comments describing how the connection should be set up as well)

Reply:
Actually, this throws an import error when `ci_prepare_provider_documentation.sh` gets run -- switching back to os.environ.get for now, though will check around later and see if there's another way to resolve.

Reply:
Hmm. Strange. Should not do that :)

Reply:
can you point me to the error :) ?

Reply:
Hi @potiuk! [This](https://github.com/apache/airflow/runs/3103723424#step:6:449) is the error I saw. I can revert my previous commit so this error will get triggered again, if that would be useful, or let me know anything else I can do to help.